<Type Name="FabricException" FullName="System.Fabric.FabricException">
  <TypeSignature Language="C#" Value="public class FabricException : Exception" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi serializable beforefieldinit FabricException extends System.Exception" />
  <TypeSignature Language="DocId" Value="T:System.Fabric.FabricException" />
  <TypeSignature Language="VB.NET" Value="Public Class FabricException&#xA;Inherits Exception" />
  <TypeSignature Language="F#" Value="type FabricException = class&#xA;    inherit Exception" />
  <AssemblyInfo>
    <AssemblyName>System.Fabric</AssemblyName>
    <AssemblyVersion>6.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>System.Exception</BaseTypeName>
  </Base>
  <Interfaces />
  <Docs>
    <summary>
      <para><span data-ttu-id="8f982-101">Service Fabric 例外の基本クラス。</span><span class="sxs-lookup"><span data-stu-id="8f982-101">The base Service Fabric exception class.</span></span></para>
    </summary>
    <remarks>
      <para><span data-ttu-id="8f982-102">基本によって定義されたプロパティだけでなく、例外が発生した正確な状況を示すために使用されるエラー コード プロパティが定義されて<see cref="T:System.Exception" />クラスです。</span><span class="sxs-lookup"><span data-stu-id="8f982-102">Defines an error code property that is used to indicate the precise circumstance that caused the exception, in addition to properties defined by the base <see cref="T:System.Exception" /> class.</span></span></para>
    </remarks>
  </Docs>
  <Members>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public FabricException ();" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Fabric.FabricException.#ctor" />
      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Fabric</AssemblyName>
        <AssemblyVersion>6.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters />
      <Docs>
        <summary>
          <para><span data-ttu-id="8f982-103">新しいインスタンスを初期化<see cref="T:System.Fabric.FabricException" />エラー コードを持つクラス<see cref="F:System.Fabric.FabricErrorCode.Unknown" />です。</span><span class="sxs-lookup"><span data-stu-id="8f982-103">Initializes a new instance of <see cref="T:System.Fabric.FabricException" /> class with error code <see cref="F:System.Fabric.FabricErrorCode.Unknown" />.</span></span></para>
        </summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public FabricException (System.Fabric.FabricErrorCode errorCode);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(valuetype System.Fabric.FabricErrorCode errorCode) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Fabric.FabricException.#ctor(System.Fabric.FabricErrorCode)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (errorCode As FabricErrorCode)" />
      <MemberSignature Language="F#" Value="new System.Fabric.FabricException : System.Fabric.FabricErrorCode -&gt; System.Fabric.FabricException" Usage="new System.Fabric.FabricException errorCode" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Fabric</AssemblyName>
        <AssemblyVersion>6.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="errorCode" Type="System.Fabric.FabricErrorCode" />
      </Parameters>
      <Docs>
        <param name="errorCode">
          <para><span data-ttu-id="8f982-104">例外に関連付けられているエラー コード。</span><span class="sxs-lookup"><span data-stu-id="8f982-104">The error code associated with the exception.</span></span></para>
        </param>
        <summary>
          <para><span data-ttu-id="8f982-105">新しいインスタンスを初期化<see cref="T:System.Fabric.FabricException" />クラスを指定したエラー コード。</span><span class="sxs-lookup"><span data-stu-id="8f982-105">Initializes a new instance of <see cref="T:System.Fabric.FabricException" /> class with a specified error code.</span></span></para>
        </summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public FabricException (string message);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Fabric.FabricException.#ctor(System.String)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String)" />
      <MemberSignature Language="F#" Value="new System.Fabric.FabricException : string -&gt; System.Fabric.FabricException" Usage="new System.Fabric.FabricException message" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Fabric</AssemblyName>
        <AssemblyVersion>6.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
      </Parameters>
      <Docs>
        <param name="message">
          <para><span data-ttu-id="8f982-106">例外の原因を説明するエラー メッセージ。</span><span class="sxs-lookup"><span data-stu-id="8f982-106">The error message that explains the reason for the exception.</span></span></para>
        </param>
        <summary>
          <para><span data-ttu-id="8f982-107">新しいインスタンスを初期化<see cref="T:System.Fabric.FabricException" />エラー コードを持つクラス<see cref="F:System.Fabric.FabricErrorCode.Unknown" />と指定したエラー メッセージ。</span><span class="sxs-lookup"><span data-stu-id="8f982-107">Initializes a new instance of <see cref="T:System.Fabric.FabricException" /> class with error code <see cref="F:System.Fabric.FabricErrorCode.Unknown" /> and a specified error message.</span></span></para>
        </summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="protected FabricException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);" />
      <MemberSignature Language="ILAsm" Value=".method familyhidebysig specialname rtspecialname instance void .ctor(class System.Runtime.Serialization.SerializationInfo info, valuetype System.Runtime.Serialization.StreamingContext context) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Fabric.FabricException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
      <MemberSignature Language="VB.NET" Value="Protected Sub New (info As SerializationInfo, context As StreamingContext)" />
      <MemberSignature Language="F#" Value="new System.Fabric.FabricException : System.Runtime.Serialization.SerializationInfo * System.Runtime.Serialization.StreamingContext -&gt; System.Fabric.FabricException" Usage="new System.Fabric.FabricException (info, context)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Fabric</AssemblyName>
        <AssemblyVersion>6.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="info" Type="System.Runtime.Serialization.SerializationInfo" />
        <Parameter Name="context" Type="System.Runtime.Serialization.StreamingContext" />
      </Parameters>
      <Docs>
        <param name="info">
          <para><span data-ttu-id="8f982-108"><see cref="T:System.Runtime.Serialization.SerializationInfo" />を含むオブジェクトがスローされた例外オブジェクト データをシリアル化します。</span><span class="sxs-lookup"><span data-stu-id="8f982-108">The <see cref="T:System.Runtime.Serialization.SerializationInfo" /> object that contains serialized object data of the exception being thrown.</span></span></para>
        </param>
        <param name="context">
          <para><span data-ttu-id="8f982-109">転送元または転送先に関するコンテキスト情報を格納する <see cref="T:System.Runtime.Serialization.StreamingContext" /> オブジェクト。</span><span class="sxs-lookup"><span data-stu-id="8f982-109">The <see cref="T:System.Runtime.Serialization.StreamingContext" /> object that contains contextual information about the source or destination.</span></span> <span data-ttu-id="8f982-110">Context パラメーターは将来使用するために予約されており、null にすることができます。</span><span class="sxs-lookup"><span data-stu-id="8f982-110">The context parameter is reserved for future use and can be null.</span></span></para>
        </param>
        <summary>
          <para><span data-ttu-id="8f982-111">新しいインスタンスを初期化<see cref="T:System.Fabric.FabricException" />コンテキストを指定して、シリアル化されたオブジェクトのデータからのクラスです。</span><span class="sxs-lookup"><span data-stu-id="8f982-111">Initializes a new instance of <see cref="T:System.Fabric.FabricException" /> class from a serialized object data, with a specified context.</span></span></para>
        </summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public FabricException (string message, Exception inner);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, class System.Exception inner) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Fabric.FabricException.#ctor(System.String,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, inner As Exception)" />
      <MemberSignature Language="F#" Value="new System.Fabric.FabricException : string * Exception -&gt; System.Fabric.FabricException" Usage="new System.Fabric.FabricException (message, inner)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Fabric</AssemblyName>
        <AssemblyVersion>6.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="inner" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="message">
          <para><span data-ttu-id="8f982-112">例外の原因を説明するエラー メッセージ。</span><span class="sxs-lookup"><span data-stu-id="8f982-112">The error message that explains the reason for the exception.</span></span></para>
        </param>
        <param name="inner">
          <para><span data-ttu-id="8f982-113">内部例外が指定されていない場合、現在の例外の原因となった例外。</span><span class="sxs-lookup"><span data-stu-id="8f982-113">The exception that is the cause of the current exception or null if no inner exception is specified.</span></span> <span data-ttu-id="8f982-114"><see cref="T:System.Exception" />クラスは、内部例外についての詳細を提供します。</span><span class="sxs-lookup"><span data-stu-id="8f982-114">The <see cref="T:System.Exception" /> class provides more details about the inner exception.</span></span></para>
        </param>
        <summary>
          <para>
            <span data-ttu-id="8f982-115">新しいインスタンスを初期化<see cref="T:System.Fabric.FabricException" />指定したエラー メッセージおよびこの例外の原因となった内部例外への参照を持つクラス。</span><span class="sxs-lookup"><span data-stu-id="8f982-115">Initializes a new instance of <see cref="T:System.Fabric.FabricException" /> class with a specified error message and a reference to the inner exception that is the cause of this exception.</span></span>
            </para>
        </summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public FabricException (string message, System.Fabric.FabricErrorCode errorCode);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, valuetype System.Fabric.FabricErrorCode errorCode) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Fabric.FabricException.#ctor(System.String,System.Fabric.FabricErrorCode)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, errorCode As FabricErrorCode)" />
      <MemberSignature Language="F#" Value="new System.Fabric.FabricException : string * System.Fabric.FabricErrorCode -&gt; System.Fabric.FabricException" Usage="new System.Fabric.FabricException (message, errorCode)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Fabric</AssemblyName>
        <AssemblyVersion>6.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="errorCode" Type="System.Fabric.FabricErrorCode" />
      </Parameters>
      <Docs>
        <param name="message">
          <para><span data-ttu-id="8f982-116">例外の原因を説明するエラー メッセージ。</span><span class="sxs-lookup"><span data-stu-id="8f982-116">The error message that explains the reason for the exception.</span></span></para>
        </param>
        <param name="errorCode">
          <para><span data-ttu-id="8f982-117">例外に関連付けられているエラー コード。</span><span class="sxs-lookup"><span data-stu-id="8f982-117">The error code associated with the exception.</span></span></para>
        </param>
        <summary>
          <para><span data-ttu-id="8f982-118">新しいインスタンスを初期化<see cref="T:System.Fabric.FabricException" />クラスは、指定したエラー メッセージとエラー コード。</span><span class="sxs-lookup"><span data-stu-id="8f982-118">Initializes a new instance of <see cref="T:System.Fabric.FabricException" /> class with specified error message and error code.</span></span></para>
        </summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public FabricException (string message, int hresult);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, int32 hresult) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Fabric.FabricException.#ctor(System.String,System.Int32)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, hresult As Integer)" />
      <MemberSignature Language="F#" Value="new System.Fabric.FabricException : string * int -&gt; System.Fabric.FabricException" Usage="new System.Fabric.FabricException (message, hresult)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Fabric</AssemblyName>
        <AssemblyVersion>6.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="hresult" Type="System.Int32" />
      </Parameters>
      <Docs>
        <param name="message">
          <para><span data-ttu-id="8f982-119">例外の原因を説明するエラー メッセージ。</span><span class="sxs-lookup"><span data-stu-id="8f982-119">The error message that explains the reason for the exception.</span></span></para>
        </param>
        <param name="hresult"><span data-ttu-id="8f982-120">例外は、周りで折り返される HResult</span><span class="sxs-lookup"><span data-stu-id="8f982-120">The HResult that the exception is wrapping around</span></span></param>
        <summary>
          <para><span data-ttu-id="8f982-121">新しいインスタンスを初期化<see cref="T:System.Fabric.FabricException" />クラスを指定したメッセージと Hresult を使用します。</span><span class="sxs-lookup"><span data-stu-id="8f982-121">Initializes a new instance of <see cref="T:System.Fabric.FabricException" /> class with specified message and Hresult.</span></span></para>
        </summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="protected FabricException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context, System.Fabric.FabricErrorCode errorCode);" />
      <MemberSignature Language="ILAsm" Value=".method familyhidebysig specialname rtspecialname instance void .ctor(class System.Runtime.Serialization.SerializationInfo info, valuetype System.Runtime.Serialization.StreamingContext context, valuetype System.Fabric.FabricErrorCode errorCode) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Fabric.FabricException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext,System.Fabric.FabricErrorCode)" />
      <MemberSignature Language="VB.NET" Value="Protected Sub New (info As SerializationInfo, context As StreamingContext, errorCode As FabricErrorCode)" />
      <MemberSignature Language="F#" Value="new System.Fabric.FabricException : System.Runtime.Serialization.SerializationInfo * System.Runtime.Serialization.StreamingContext * System.Fabric.FabricErrorCode -&gt; System.Fabric.FabricException" Usage="new System.Fabric.FabricException (info, context, errorCode)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Fabric</AssemblyName>
        <AssemblyVersion>6.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="info" Type="System.Runtime.Serialization.SerializationInfo" />
        <Parameter Name="context" Type="System.Runtime.Serialization.StreamingContext" />
        <Parameter Name="errorCode" Type="System.Fabric.FabricErrorCode" />
      </Parameters>
      <Docs>
        <param name="info">
          <para><span data-ttu-id="8f982-122"><see cref="T:System.Runtime.Serialization.SerializationInfo" />を含むオブジェクトがスローされた例外オブジェクト データをシリアル化します。</span><span class="sxs-lookup"><span data-stu-id="8f982-122">The <see cref="T:System.Runtime.Serialization.SerializationInfo" /> object that contains serialized object data of the exception being thrown.</span></span></para>
        </param>
        <param name="context">
          <para><span data-ttu-id="8f982-123">転送元または転送先に関するコンテキスト情報を格納する <see cref="T:System.Runtime.Serialization.StreamingContext" /> オブジェクト。</span><span class="sxs-lookup"><span data-stu-id="8f982-123">The <see cref="T:System.Runtime.Serialization.StreamingContext" /> object that contains contextual information about the source or destination.</span></span> <span data-ttu-id="8f982-124">Context パラメーターは将来使用するために予約されており、null にすることができます。</span><span class="sxs-lookup"><span data-stu-id="8f982-124">The context parameter is reserved for future use and can be null.</span></span></para>
        </param>
        <param name="errorCode">
          <para><span data-ttu-id="8f982-125">例外に関連付けられているエラー コード。</span><span class="sxs-lookup"><span data-stu-id="8f982-125">The error code associated with the exception.</span></span></para>
        </param>
        <summary>
          <para><span data-ttu-id="8f982-126">新しいインスタンスを初期化<see cref="T:System.Fabric.FabricException" />指定のコンテキストとエラー コードを含む、シリアル化されたオブジェクトのデータからのクラスです。</span><span class="sxs-lookup"><span data-stu-id="8f982-126">Initializes a new instance of <see cref="T:System.Fabric.FabricException" /> class from a serialized object data, with specified context and error code.</span></span></para>
        </summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public FabricException (string message, Exception inner, System.Fabric.FabricErrorCode errorCode);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, class System.Exception inner, valuetype System.Fabric.FabricErrorCode errorCode) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Fabric.FabricException.#ctor(System.String,System.Exception,System.Fabric.FabricErrorCode)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, inner As Exception, errorCode As FabricErrorCode)" />
      <MemberSignature Language="F#" Value="new System.Fabric.FabricException : string * Exception * System.Fabric.FabricErrorCode -&gt; System.Fabric.FabricException" Usage="new System.Fabric.FabricException (message, inner, errorCode)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Fabric</AssemblyName>
        <AssemblyVersion>6.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="inner" Type="System.Exception" />
        <Parameter Name="errorCode" Type="System.Fabric.FabricErrorCode" />
      </Parameters>
      <Docs>
        <param name="message">
          <para><span data-ttu-id="8f982-127">例外の原因を説明するエラー メッセージ。</span><span class="sxs-lookup"><span data-stu-id="8f982-127">The error message that explains the reason for the exception.</span></span></para>
        </param>
        <param name="inner">
          <para><span data-ttu-id="8f982-128">内部例外が指定されていない場合、現在の例外の原因となった例外。</span><span class="sxs-lookup"><span data-stu-id="8f982-128">The exception that is the cause of the current exception or null if no inner exception is specified.</span></span> <span data-ttu-id="8f982-129"><see cref="T:System.Exception" />クラスは、内部例外についての詳細を提供します。</span><span class="sxs-lookup"><span data-stu-id="8f982-129">The <see cref="T:System.Exception" /> class provides more details about the inner exception.</span></span></para>
        </param>
        <param name="errorCode">
          <para><span data-ttu-id="8f982-130">例外に関連付けられているエラー コード。</span><span class="sxs-lookup"><span data-stu-id="8f982-130">The error code associated with the exception.</span></span></para>
        </param>
        <summary>
          <para>
            <span data-ttu-id="8f982-131">新しいインスタンスを初期化<see cref="T:System.Fabric.FabricException" />クラスを指定したエラー メッセージ、この例外と、指定されたエラー コードの原因となった内部例外への参照。</span><span class="sxs-lookup"><span data-stu-id="8f982-131">Initializes a new instance of <see cref="T:System.Fabric.FabricException" /> class with a specified error message, a reference to the inner exception that is the cause of this exception, and a specified error code.</span></span>
            </para>
        </summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public FabricException (string message, Exception inner, int hresult);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, class System.Exception inner, int32 hresult) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Fabric.FabricException.#ctor(System.String,System.Exception,System.Int32)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, inner As Exception, hresult As Integer)" />
      <MemberSignature Language="F#" Value="new System.Fabric.FabricException : string * Exception * int -&gt; System.Fabric.FabricException" Usage="new System.Fabric.FabricException (message, inner, hresult)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Fabric</AssemblyName>
        <AssemblyVersion>6.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="inner" Type="System.Exception" />
        <Parameter Name="hresult" Type="System.Int32" />
      </Parameters>
      <Docs>
        <param name="message">
          <para><span data-ttu-id="8f982-132">例外の原因を説明するエラー メッセージ。</span><span class="sxs-lookup"><span data-stu-id="8f982-132">The error message that explains the reason for the exception.</span></span></para>
        </param>
        <param name="inner">
          <para><span data-ttu-id="8f982-133">内部例外。</span><span class="sxs-lookup"><span data-stu-id="8f982-133">The inner exception.</span></span></para>
        </param>
        <param name="hresult"><span data-ttu-id="8f982-134">例外は、周りで折り返される HResult</span><span class="sxs-lookup"><span data-stu-id="8f982-134">The HResult that the exception is wrapping around</span></span></param>
        <summary>
          <para><span data-ttu-id="8f982-135">新しいインスタンスを初期化<see cref="T:System.Fabric.FabricException" />した指定したメッセージ、内部例外を HResult。</span><span class="sxs-lookup"><span data-stu-id="8f982-135">Initializes a new instance of <see cref="T:System.Fabric.FabricException" /> class with specified message, inner exception and HResult.</span></span></para>
        </summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName="ErrorCode">
      <MemberSignature Language="C#" Value="public System.Fabric.FabricErrorCode ErrorCode { get; }" />
      <MemberSignature Language="ILAsm" Value=".property instance valuetype System.Fabric.FabricErrorCode ErrorCode" />
      <MemberSignature Language="DocId" Value="P:System.Fabric.FabricException.ErrorCode" />
      <MemberSignature Language="VB.NET" Value="Public ReadOnly Property ErrorCode As FabricErrorCode" />
      <MemberSignature Language="F#" Value="member this.ErrorCode : System.Fabric.FabricErrorCode" Usage="System.Fabric.FabricException.ErrorCode" />
      <MemberType>Property</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Fabric</AssemblyName>
        <AssemblyVersion>6.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <ReturnValue>
        <ReturnType>System.Fabric.FabricErrorCode</ReturnType>
      </ReturnValue>
      <Docs>
        <summary>
          <para><span data-ttu-id="8f982-136">エラー コード パラメーターを取得します。</span><span class="sxs-lookup"><span data-stu-id="8f982-136">Gets the error code parameter.</span></span></para>
        </summary>
        <value>
          <para><span data-ttu-id="8f982-137">エラー コードに関連付けられている、<see cref="T:System.Fabric.FabricException" />例外。</span><span class="sxs-lookup"><span data-stu-id="8f982-137">The error code associated with the <see cref="T:System.Fabric.FabricException" /> exception.</span></span></para>
        </value>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
  </Members>
</Type>